src/pages/home/Homepage.tsx

```
@@ -1,36 +1,66 @@
-import BottomSheet from "@/components/bottom-sheet/BottomSheet";
-import Lock from "@/components/lock/Lock";
-import UnLock from "@/components/lock/UnLock";
+import LoadingDots from "@/components/loading/loading";
 
-type BottomSheetType = "lock" | "unlock" | "clear";
+import FirstPlant from "@/components/home/FirstPlant";
+import SecondPlant from "@/components/home/SecondPlant";
+import ThirdPlant from "@/components/home/ThirdPlant";
+import FourthPlant from "@/components/home/FourthPlant";
+
+import "@/styles/home/swiper.css";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)